### PR TITLE
Do not use python-meh

### PR DESCRIPTION
**Pull request by vojtechtrefny**
python-meh shouldn't be used outside installer environment -- https://bugzilla.redhat.com/show_bug.cgi?id=1358066

### CODE REVIEW

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

File: blivet-gui.spec [hunk 1]
```diff
@@ -63,6 +63,7 @@ fi
 %{_bindir}/blivet-gui
 %{_bindir}/blivet-gui-daemon
 %{_localstatedir}/log/blivet-gui
+/etc/libreport/events.d/blivet-gui_event.conf
```
Comment (vpodzime) [inline blivet-gui.spec]: This should use the macro for ``/etc`` and *libreport* should be added to requires.

**Comment by vpodzime** [comment]
Looks good to me otherwise.

File: blivet-gui.spec (hunk 1 above)
Comment (vojtechtrefny) [inline blivet-gui.spec]: Fixed, thanks.

**Pull request merged**